components/products/products.html

```diff
@@ -1 +1,23 @@
-<p>Products works</p>
\ No newline at end of file
+<div class="container mt-5 m-auto p-5">
+    <div class="row m-auto mb-5 pb-3">
+        <h1 class="display-2 text-center">Products List</h1>
+    </div>
+    <div class="row row-cols-1 row-cols-md-3 g-4">
+        <div ng-repeat="prod in $ctrl.arrayProducts" class="col">
+            <div class="card h-100 border-dark mb-3">
+                <!-- <img src="..." class="card-img-top" alt="..."> -->
+                <div class="card-body">
+                    <h5 class="card-title">Product: {{prod.name}}</h5>
+                    <p ng-if="prod.description != ''" class="card-text"><i>Description:</i> {{prod.description}}</p>
+                    <p ng-else class="card-text"><i>No available description</i></p>
```

Comment:
I don't think we really need the elif.js, this ng-else could be ng-if="!prod.description". As simple as that :)